### ISSUE TITLE
pcapy 0.11.5 missing from PyPI

### ISSUE THREAD

**Issue by N7-Revenant** [CoreSecurity/pcapy#62]
Hi,

I've noticed, that pcapy 0.11.5 was released several weeks ago, but new version still missing from PyPI. It's a little inconvenient, since version 0.11.4 has a memory leak: https://github.com/helpsystems/pcapy/commit/20a533f014344f25bc859fef016f2766af008fdc, but it was patched in 0.11.5

Is it possible to add pcapy 0.11.5 to PyPI?

**Comment by gpotter2** [comment]
Yeah.. I think we can all agree that the project is abandoned at this point.

That's what you get with forced changes of ownership.. https://github.com/helpsystems/pcapy/pull/60#issuecomment-483821969
